drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -408,6 +433,24 @@ def _get_item_definition(self, item_id):
         """
         return next(i for i in self.data['items'] if i['id'] == item_id)
 
+    def _get_valid_item_zones(self, item_id):
+        """
+        Returns a list of the zones that are valid options for the item. If
+        we have an existing configured list of options, return that. If we
+        have an existing single item, return that encapsulated in a list.
+        If we don't have either, which would be weird, return an empty list.
```

Comment:
@haikuginger Please add a line break and an empty line after the first sentence of the docstring (not all docstrings in this file currently do this, but for newly added code we should try and follow edX's [Python Guidelines](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings)).

For the rest of the docstring, it would be good to make the terminology more specific (to better reflect the fact that the method deals with zones, not options or items):

> If the item is associated with a list of zones, return that. If the item is associated with a single zone, return that encapsulated in a list.